components/ui/Tag.jsx

```diff
@@ -0,0 +1,12 @@
+function Tag(props) {
+  return (
+    <a
+      className={`h-fit rounded-4.75 bg-light-grey py-0.75 px-2.5 font-Neufile text-lg font-normal tracking-spacing-lg text-grey ${props.style}`}
+      href={props.link}
+    >
+      {props.children}
```

Comment:
I think it is not a good idea to use chidren when receing one props, I suggest you use something like title or name

Reply:
That's a react universal property kindly google it

Reply:
Sure, you are right.